Unfortunately the version as written falls over when trying to build LLVM:

```
CMake Error at cmake/modules/AddLLVM.cmake:1408 (add_dependencies):
  The dependency target "clang" of target "check-all" does not exist.
Call Stack (most recent call first):
  CMakeLists.txt:977 (add_lit_target)


CMake Error at cmake/modules/AddLLVM.cmake:1408 (add_dependencies):
  The dependency target "clang" of target "check-llvm-spirv" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1429 (add_lit_target)
  projects/SPIRV-LLVM-Translator/test/CMakeLists.txt:69 (add_lit_testsuite)


-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    COMPILER_RT_INCLUDE_TESTS
    COMPILER_RT_USE_LIBCXX
```

I'd assumed the `-DCMAKE_CXX_FLAGS=-static-libstdc++` and `-DCOMPILER_RT_USE_LIBCXX=OFF` would avoid that problem, but seems not -- any suggestions for what could be wrong?  (I can't see obvious differences with the LDC CI setup.)

--- a/snap/snapcraft.yaml
+++ b/snap/snapcraft.yaml
@@ -97,19 +97,27 @@ parts:
     - llvm
 
   llvm:
-    source: https://github.com/ldc-developers/llvm.git
-    source-tag: ldc-v9.0.0
+    source: https://github.com/ldc-developers/llvm-project.git
+    source-tag: ldc-v9.0.1
     source-type: git
     plugin: cmake
-    configflags:
-    - -DCMAKE_BUILD_TYPE=Release
-    - -DCOMPILER_RT_INCLUDE_TESTS=OFF
-    - -DLLVM_BINUTILS_INCDIR=/usr/include
-    - -DLLVM_TARGETS_TO_BUILD=AArch64;ARM;Mips;MSP430;NVPTX;PowerPC;X86
-    - -DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD=RISCV;WebAssembly
-    - -DLLVM_ENABLE_TERMINFO=OFF
-    - -DLLVM_ENABLE_LIBEDIT=OFF
-    - -DLLVM_INCLUDE_TESTS=OFF
+    override-build: |
+      cmake ../src/llvm \
+        -DCMAKE_INSTALL_PREFIX= \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_CXX_FLAGS=-static-libstdc++ \
+        -DCOMPILER_RT_INCLUDE_TESTS=OFF \
+        -DCOMPILER_RT_USE_LIBCXX=OFF \
+        -DLLVM_BINUTILS_INCDIR=/usr/include \
+        -DLLVM_TARGETS_TO_BUILD='AArch64;ARM;Mips;MSP430;NVPTX;PowerPC;RISCV;WebAssembly;X86' \
+        -DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD='AVR' \
+        -DLLVM_ENABLE_PROJECTS='compiler-rt;lld' \
+        -DLLVM_ENABLE_TERMINFO=OFF \
+        -DLLVM_ENABLE_LIBEDIT=OFF \
+        -DLLVM_INCLUDE_TESTS=OFF \
+        -GNinja
+      ninja
+      DESTDIR=../install ninja install
     stage:
     - -*
     prime:
@@ -117,6 +125,7 @@ parts:
     build-packages:
     - binutils-dev
     - g++
+    - ninja-build
 
   ctest-setup:
     plugin: nil

